source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -569,13 +569,13 @@ function Initialize-ProxySettings() {
 		#system proxy		
 		if ($useDefaultProxy) {
 			# The system proxy should be provided through an environment variable, which has been used to initialize $proxyHost
-			if ($proxyUri -ne $null) {
+			if ($octopusProxyUri -ne $null) {
 				# isWindows was introduced in PSCore and is not available for standard PowerShell
-				if (-not (Test-Path variable:IsWindows) -or $isWindows){
+				if (-not ((Test-Path variable:IsWindows) -or ($isWindows))){
```

Comment:
This is an unrelated bug that I noticed while debugging this code.

Reply:
Nice find! I'd be keen to see this tracked as a separate issue though, that outlines what the bug was to make it easier for future Tom and future Emily. I've only been here a couple of weeks but have already had to refer back to issues related to this check.